packages/features/form-builder/schema.ts

```diff
@@ -318,6 +338,36 @@ export const fieldTypesSchemaMap: Partial<
       });
     },
   },
+  textarea: {
+    preprocess: ({ response }) => {
+      return response.trim();
+    },
+    superRefine: ({ field, response, ctx, m }) => {
+      const fieldTypeConfig = fieldTypesConfigMap[field.type];
+      const value = response ?? "";
+      const maxLength = field.maxLength ?? fieldTypeConfig.supportsLengthCheck?.maxLength;
+      const minLength = field.minLength ?? 0;
```

Comment:
Ensured that `maxLength` and `minLength` are initialized correctly and use them only and not field props.